modules/admin/src/main/java/org/motechproject/admin/internal/service/impl/SettingsServiceImpl.java

```diff
@@ -85,6 +85,12 @@ public AdminSettings getSettings() {
             securityOptions.add(minPasswordLengthOption);
             SettingsOption passwordValidatorOption = ParamParser.parseParam(ConfigurationConstants.PASSWORD_VALIDATOR, motechSettings.getPasswordValidator());
             securityOptions.add(passwordValidatorOption);
+            SettingsOption passwordResetOption = ParamParser.parseParam(ConfigurationConstants.PASSWORD_RESET_DAYS, motechSettings.getNumberOfDaysToChangePassword());
```

Comment:
It would be best if the reminder checkbox was above the reminder days input. Not sure where does the current the order come from.